expyfun/stimuli/_tracker.py

```diff
@@ -254,6 +262,31 @@ def respond(self, correct):
             self._x = np.append(self._x, self._x[-1] +
                                 self._current_step_size_up)
 
+        if bound:
+            if self._x_min is not None:
```

Comment:
wait, it can be `None`? I thought it was set to `np.inf` or `-np.inf` in the constructor